core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -657,7 +685,7 @@ ByteBuffer getKey(K key) {
 
         Chunk<K, V> c = findChunk(key);
         Chunk.LookUp lookUp = c.lookUp(key);
-        if (lookUp == null || lookUp.handle == null || lookUp.entryIndex == -1) {
+        if (lookUp == null || lookUp.valueSlice == null || lookUp.entryIndex == -1) {
             return null;
         }
         return c.readKey(lookUp.entryIndex).slice();
```

Comment:
Where was/is it used? Why do we have to add slice() at the end here? Can you please add some comment?

Reply:
This method is used in `getKeyTransformation` which is currently not used, though I assume it is used in Druid.